boards/saml21-xpro/Makefile.include

```diff
@@ -1,3 +1,5 @@
 CFLAGS += -D__SAML21J18A__
 
+USEMODULE += periph_uart_hw_fc
```

Comment:
Please add this line in boards/saml21-xpro/Makefile.dep

Reply:
I'll remove it instead. There is no need to enable this module by default on SAML21-XPRO right ?
I dunno why this line is here...

Reply:
Not sure if this line should still be there ?

Reply:
This line is my only remaining concern: can we move it to the board `Makefile.dep` or just remove it ?

Reply:
This one is still here ?! Damn, I'm sorry I'll remove it